src/test/resources/db.changelog/changes/01-create-books-table.yaml

```
@@ -0,0 +1,55 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-books-table
+      author: mateacademy
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+
+              - column:
+                  name: title
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+
+              - column:
+                  name: author
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+
+              - column:
+                  name: isbn
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+
+              - column:
+                  name: price
+                  type: decimal(19,2)
+                  constraints:
+                    nullable: false
+
+              - column:
+                  name: description
+                  type: varchar(1000)
+
+              - column:
+                  name: cover_image
+                  type: varchar(500)
+
+              - column:
+                  name: isDeleted
```

have you run and test your project?
```suggestion
                  name: is_deleted
```